assets/src/dashboard/app/views/editorSettings/publisherLogo/index.js

```diff
@@ -46,21 +48,49 @@ const TEXT = {
     'web-stories'
   ),
   SUBMIT: __('Upload', 'web-stories'),
+  ARIA_LABEL: __('Click to upload a new logo', 'web-stories'),
+  HELPER_UPLOAD: __('You can also drag your logo here', 'web-stories'),
 };
 
-// eslint-disable-next-line no-unused-vars
 function PublisherLogoSettings({ onUpdatePublisherLogo, publisherLogos }) {
+  const onSubmitNewFile = useCallback(
+    async (files) => {
+      const resources = await Promise.all(
+        files.map(async (file) => ({
+          localResource: await getResourceFromLocalFile(file),
+          file,
+        }))
+      );
+      onUpdatePublisherLogo({ newPublisherLogos: resources });
+    },
+    [onUpdatePublisherLogo]
+  );
+
+  const onSubmitDeleteFile = useCallback(
+    (_, fileData) => {
+      onUpdatePublisherLogo({ deleteLogo: fileData });
+    },
+    [onUpdatePublisherLogo]
+  );
+
   return (
     <SettingForm>
       <SettingHeading htmlFor="publisherLogo">
         {TEXT.SECTION_HEADING}
       </SettingHeading>
       <div>
         <FileUploadHelperText>{TEXT.CONTEXT}</FileUploadHelperText>
-        <UploadContainer>
-          <p>{'Upload Placeholder'}</p>
-          <button>{TEXT.SUBMIT}</button>
-        </UploadContainer>
+        <UploadContainer
+          acceptableFormats={['.jpg', '.jpeg', '.png', '.gif']}
```

Comment:
Super nit/question, can this list of formats be the default?

Reply:
sure! added gif to defaults and removed the prop here since it'd be redundant